_gsdocs-projects/2020/project_AllpixSquared.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Allpix Squared
+project: AllpixSquared
+layout: default
+logo: AllpixSquared.png
```

Comment:
Image should be named following project-logo.png

Reply:
Done - had to also rename the file in old GSOC proposals though.

Reply:
In general we make a new copy if an old file was available - not to affect old pages. If you did it properly and grep'ed for the old file name and changed to the new name - it's fine I guess

Reply:
I did - and even used `git mv` for renaming... :)